### PR TITLE
Update Discord & MS Teams install doc

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update Discord install doc
- Update MS Teams install doc

PreviewURL: https://fix-install-docs.botkube-docs.pages.dev/

## Testing

In docs I specified the 0.13 version but I ran the same command with the 0.13.0-rc.1.
```bash
export CLUSTER_NAME=dev
export ALLOW_KUBECTL=true

helm install --version v0.13.0-rc.1 botkube --namespace botkube --create-namespace \
--set communications.default-group.discord.enabled=true \
--set communications.default-group.discord.channels.default.id=${DISCORD_CHANNEL_ID} \
--set communications.default-group.discord.botID=${DISCORD_BOT_ID} \
--set communications.default-group.discord.token=${DISCORD_TOKEN} \
--set settings.clusterName=${CLUSTER_NAME} \
--set executors.kubectl-read-only.kubectl.enabled=${ALLOW_KUBECTL} \
--set image.tag=v0.13.0-rc.1 \
botkube/botkube
```

## Related issue(s)

- https://github.com/kubeshop/botkube/issues/595

### CODE REVIEW

File: content/installation/Discord/_index.md [hunk 1]
```diff
@@ -87,21 +99,24 @@ Follow the first 4 mins of this [Video Tutorial](https://youtu.be/8o25pRbXdFw) t
...
+  export ALLOW_KUBECTL={allow_kubectl}
+
+  helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
+  --set communications.default-group.discord.enabled=true \
+  --set communications.default-group.discord.channels.default.id=${DISCORD_CHANNEL_ID} \
+  --set communications.default-group.discord.botID=${DISCORD_BOT_ID} \
+  --set communications.default-group.discord.token=${DISCORD_TOKEN} \
+  --set settings.clusterName=${CLUSTER_NAME} \
+  --set executors.kubectl-read-only.kubectl.enabled=${ALLOW_KUBECTL} \
+  --set image.tag=v0.13.0 \
```
Comment (pkosiec) [inline content/installation/Discord/_index.md:119]: Question: Do we want to define override for image tag, if the default value inside `v0.13.0` Helm chart will have proper one already set? I think this is not necessary 🤔

Comment (mszostok) [reply content/installation/Discord/_index.md:119]: good point 👍 totally agree :)

File: content/installation/Teams/_index.md [hunk 2]
```diff
@@ -170,22 +182,21 @@ We will use this TLS secret while deploying the BotKube backend.
...
-     A Full Helm chart parameters list is documented [here](/configuration/helm-chart-parameters)
+  Configuration syntax is explained [here](/configuration). A Full Helm chart parameters list is documented [here](/configuration/helm-chart-parameters).
 
-    Send **@BotKube ping** in the channel to see if BotKube is running and responding.
+- Send **@BotKube ping** in the channel to see if BotKube is running and responding.
 
-    With the default configuration, BotKube will watch all the resources in all the namespaces for _create_, _delete_ and _error_ events.<br>
-    If you wish to monitor only specific resources, follow the steps given below:
+  With the default configuration, BotKube will watch all the resources in all the namespaces for _create_, _delete_ and _error_ events.<br>
+  If you wish to monitor only specific resources, follow the steps given below:
```
Comment (pkosiec) [inline content/installation/Teams/_index.md:190]:
```suggestion
  With the default configuration, BotKube will watch all the resources in all the namespaces for _create_, _delete_ and _error_ events.

    If you wish to monitor only specific resources, follow the steps given below:
```

**Review by pkosiec — APPROVED** [review]
LGTM

**Pull request merged**